src/screens/DomainScreen/elements/Header/elements/CredderInfoGroup.js

```diff
@@ -66,7 +66,7 @@ const styles = StyleSheet.create({
     fontFamily: fonts.inter[400],
     fontSize: RFValue(14),
     lineHeight: RFValue(17),
-    color: COLORS.white2
+    color: COLORS.white
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color value has been changed from `COLORS.white2` to `COLORS.white`. Ensure that this change doesn't affect the readability and contrast of the text against its background. If `COLORS.white2` was a lighter shade, it might have been used to provide better contrast.
```diff
-    color: COLORS.white2
+    color: COLORS.white
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->